roles/as_code/lib/lead_engineer.rb

```diff
@@ -2,7 +2,60 @@
 require_relative './shared_expectations/delivery_lead_expectations'
 
 JobSpec::Role.definition 'Lead Engineer' do
-  description 'Our Lead Engineers take responsibility for the successful delivery of software products and services. Experts in both software craftsmanship and agile delivery methods, Lead Engineers nurture engineering teams and are trusted partners to our customers.'
+  description <<~DESCRIPTION
+    Our Lead Engineers turn our customer’s needs into roadmaps, coach teams to deliver, are cutting the best code of their career, still learn everyday, and constantly strive to be nice humans.
+
+    ## What does the job entail?
+
+    We primarily write and deliver custom software to our customers. Before a project kicks off we will have a number of sessions where we begin to understand the customer’s business and what they are looking to achieve. We then formulate a roadmap for them, potentially a business case too, before starting delivery.
+
+    We build solutions in a variety of languages and platforms, historically Ruby on Rails, we tend to use React, JavaScript, Elixir, and Clojure these days. We will usually be responsible for setting up a customer’s infrastructure either on AWS with tools like Ansible and Terraform. Sometimes we opt for a Platform as a Service like Heroku. We will make sure there is a continuous delivery pipeline running our automated test suites before deploying to a staging environment, and a manual button for going to production.
```

Comment:
Continuous Delivery.
Still find the prescription of the pipeline down to the manual push-button for production strange in this context